javascript/javascript1/week4/palindromeSubstring.js

```diff
@@ -1,4 +1,38 @@
 //A palindrome is a word, number, phrase, or other sequence of characters which reads the same backward as forward. eg: madam
+
+function longestPalindrome1(str) {
+  const entireString = str.toLowerCase();
+  if (str.length < 2) {
+    return "";
+  }
+
+  let palindrome = "";
+
+  for (let i = 0; i < entireString.length; i++) {
+    for (let j = 1; j <= entireString.length - i; j++) {
```

Comment:
I believe you wanted to do this...you have to generate all the substrings  `string[i][j]` where  0<= i < n and   i < j < n where n is the length of the string. 
```suggestion
    for (let j = i; j <= entireString.length ; j++) {
```